packages/react-integration/demo-app-ts/src/components/demos/AlertGroupDemo/AlertGroupDemo.tsx

```diff
@@ -36,7 +45,7 @@ export class AlertGroupDemo extends React.Component<{}, AlertGroupDemoState> {
       this.setState({ alerts: [...this.state.alerts, ...incomingAlerts] });
     };
     const getUniqueId = () => new Date().getTime();
-    const btnClasses = ['pf-v5-c-button', 'pf-m-secondary'].join(' ');
+    const btnClasses = css(styles.button, styles.modifiers.secondary);
```

Comment:
you use `.join()` in other places where you're doing this. is that needed?

```suggestion
    const btnClasses = css(styles.button, styles.modifiers.secondary).join(' ');
```

Reply:
I am using the `css` function here, which is a wrapper which "joins args into a className string".
So `[buttonStyles.button, buttonStyles.modifiers.secondary].join(' ');` is the same as `css(styles.button, styles.modifiers.secondary)`.

We could probably use the `css(classnames)` function everywhere, instead of `[classnames].join(" ")`, but the result is the same.